lambdas/data-migration1/tests/test-async-operations.js

```diff
@@ -26,8 +26,8 @@ const testDbUser = 'postgres';
 const generateFakeAsyncOperation = (params) => ({
   id: uuid(),
   description: 'unittest async operation',
+  output: '\"Index from database complete\"',
```

Comment:
Was there a functional reason for this change?   `output` should be able to be any JSON parseable string (e.g. `description: 'The result of the operation, stored as JSON'), although that's not directly consistent with the postgres comment.